src/tx-format/transaction.md

```diff
@@ -131,15 +131,25 @@ Creates a contract with contract ID as computed [here](../identifiers/contract-i
 ## TransactionMint
 
 The transaction is created by the block producer and is not signed. Since it is not usable outside of block creation or execution, all fields must be fully set upon creation without any zeroing.
-
-| name           | type                         | description                                          |
-|----------------|------------------------------|------------------------------------------------------|
-| `txPointer`    | [TXPointer](./tx-pointer.md) | The location of the `Mint` transaction in the block. |
-| `outputsCount` | `uint8`                      | Number of outputs.                                   |
-| `outputs`      | [Output](./output.md)`[]`    | List of outputs.                                     |
+This means that the transaction ID must also include the correct `txPointer` value, not zeroed out.
+
+| name           | type                         | description                                               |
+|----------------|------------------------------|-----------------------------------------------------------|
+| `mintAmount`   | `uint64`                     | The amount of funds being minted within this transaction. |
+| `mintAssetId`  | `byte[32]`                   | The asset ID of funds minted within this transaction.     |
```

Comment:
It may make sense to have an array of the mint assets. In the case if we want to support paying in different assets=)

Reply:
I thought about that, and it kind of made me wonder if it'd be better to just replace the concept of a mint transaction with a mint input that has special spending rules.

Eventually that could be opened up later to enable things like flashloans etc.

Reply:
decided to keep it simple with a single assetID